src/gffread/config.vsh.yaml

```
@@ -0,0 +1,396 @@
+name: gffread
+namespace: gffread
+description: Validate, filter, convert and perform various other operations on GFF files.
+keywords: [gff, conversion, validation, filtering]
+links:
+  homepage: https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread
+  documentation: https://ccb.jhu.edu/software/stringtie/gff.shtml#gffread
+  repository: https://github.com/gpertea/gffread
+references: 
+  doi: 10.12688/f1000research.23297.2
+license: MIT
+requirements:
+  commands: [ gffread ]
+argument_groups:
+  - name: Inputs
+    arguments:
+      - name: --input
+        type: file 
+        direction: input
+        description: |
+          A reference file in either the GFF3, GFF2 or GTF format.
+        required: true
+        example: annotation.gff
+      - name: --chr_mapping
+        alternatives: -m
+        type: file
+        direction: input
+        description: |
+          <chr_replace> is a name mapping table for converting reference sequence names, 
+          having this 2-column format: <original_ref_ID> <new_ref_ID>.
+      - name: --seq_info
+        alternatives: -s
+        type: file
+        direction: input
+        description: |
+          <seq_info.fsize> is a tab-delimited file providing this info for each of the mapped 
+          sequences: <seq-name> <seq-length> <seq-description> (useful for -A option with 
```

Could you edit the descriptions such that it refers to the new argument names?

```suggestion
          sequences: <seq-name> <seq-length> <seq-description> (useful for --description option with 
```